website/docs/r/cdn_frontdoor_security_policy.html.markdown

```diff
@@ -71,8 +71,6 @@ resource "azurerm_cdn_frontdoor_security_policy" "example" {
 
 ## Arguments Reference
 
-The following arguments are supported:
```

Comment:
why's this been removed/can we add this back in?

Reply:
In a previous PR you told me to remove this, so I removed it from all of the documentation since I thought that was the new standard. Fixed.